.github/workflows/phpunit.yml

```diff
@@ -83,14 +83,15 @@ jobs:
         options: --health-cmd=pg_isready --health-interval=10s --health-timeout=5s --health-retries=3
 
       mssql:
-        image: mcr.microsoft.com/mssql/server:2019-CU10-ubuntu-20.04
+        image: mcr.microsoft.com/mssql/server:2019-CU28-ubuntu-20.04
         env:
           SA_PASSWORD: 1Secure*Password1
           ACCEPT_EULA: Y
           MSSQL_PID: Developer
+          MSSQL_ENCRYPT: optional
```

Comment:
Why optional?

Reply:
I’m not entirely sure if GitHub Actions in this repo uses a valid certificate for connecting to MSSQL. Therefore, as a precaution, I’ve set the `MSSQL_ENCRYPT` option to `optional`. This setting allows us to have an encrypted connection if a valid certificate is present, but it also ensures that we can continue to connect to the database even if a valid certificate is not available. 
I previously tested these changes on repo **codeigniter-dea-rule**, and all tests ran successfully.

https://github.com/datamweb/codeigniter-dea-rule/blob/1e128372ed0c1ee6a7a6aff3fedf7c8463815f80/.github/workflows/phpunit.yml#L90